pyproject.toml

```diff
@@ -29,6 +29,9 @@ nautobot = "~1.2.1"
 boto3 = { version = "~1.19.5", optional = true }
 # HashiCorp Vault API Client (always keep in sync w/ dev dependency)
 hvac = { version = "~0.11.0", optional = true }
+# Thycotic Secret Server (always keep in sync w/ dev dependency)
+python-tss-sdk = {version = ">=1.0.1,<2.0", optional = true}
```

Comment:
It looks like the next version of python-tss-sdk (which will remove the `dataclasses` dependency) is also going to drop support for python 3.6 (understandably) but as Nautobot 1.2.x still supports python 3.6, it might be desirable to add a `python = ">=3.7"` to this dependency, and document this restriction in the README.

Does this PR need to be blocked until a new release of python-tss-sdk is published?

Reply:
I just released the latest version of the `python-tss-sdk` (1.1.0) to PyPI. I hope this helps the PR author to move this along. Great work @jifox.